### PR TITLE
Feat/check intervals consistency

### PR DESCRIPTION
**Pull request by no-chris**

### CODE REVIEW

File: src/parser/filters/checkIntervalsConsistency.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { hasAll } from '../../helpers/hasElement';
+
+const allForbiddenCombos = [
+	//['2',  '3'], // "2" modifier yields a ninth interval and not a second (eg, C2 = Cadd9)
+	['3',  'b3'],
+	//['3',  '4'], // valid in the Real Book: F#7SUS(add 3)
+	['5',  'b5'],
+	['5',  '#5'],
+	['b7', 'bb7'],
+	//['7',  'bb7'], // valid in the Real Book: C°7(add MA7)
```
Comment (Marr11317) [inline src/parser/filters/checkIntervalsConsistency.js:10]: it definetely is a valid extension. 
This bb7 is often considered as a 6. For a more detailed explanation, diminished chords may contain any note in the diminished chord a tone above. So a `C°7` may contain any note in `D°7`.

File: src/parser/filters/checkIntervalsConsistency.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { hasAll } from '../../helpers/hasElement';
+
+const allForbiddenCombos = [
+	//['2',  '3'], // "2" modifier yields a ninth interval and not a second (eg, C2 = Cadd9)
```
Comment (Marr11317) [inline src/parser/filters/checkIntervalsConsistency.js:4]: what about 2 and 9? 4 and 11? etc.?

**Review by Marr11317 — COMMENTED** [review]
Looks good! I have left a few comments, mostly for some clarification.

File: src/parser/filters/checkIntervalsConsistency.js (hunk 2 above)
Comment (no-chris) [reply src/parser/filters/checkIntervalsConsistency.js]: Done

**Comment by no-chris** [comment]
Thanks a lot for the review 👍

**Pull request merged**